File: src/LearningApp.Service/LearningApp.Service.API.Contracts/Users/Responses/UserInfoResponse.cs
using System;
using LearningApp.Service.API.Contracts.Users.Common;

namespace LearningApp.Service.API.Contracts.Users.Responses
{
	public class UserInfoResponse
	{
		public long Id { get; set; }

		public string Email { get; set; }

		public string Username { get; set; }

		public PermissionLevel PermissionLevel { get; set; }

		public Language Language { get; set; }

		public DateTimeOffset RegistrationDate { get; set; }
	}
}

File: src/LearningApp.Android/app/src/main/java/com/example/learningappandroid/api/contracts/users/responses/UserInfoResponse.kt
package com.example.learningappandroid.api.contracts.users.responses

import com.example.learningappandroid.api.contracts.questions.common.LastSubmissionsInfo
import com.example.learningappandroid.api.contracts.users.common.Language
import com.example.learningappandroid.api.contracts.users.common.PermissionLevel

data class UserInfoResponse(
	val id: Long,
	val email: String,
	val username: String,
	val permissionLevel: PermissionLevel,
	val language: Language,
	val registrationDate: String,
	val lastSubmissionsInfo: LastSubmissionsInfo
)